content/blog/2022/04/secure-supply-chain.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Secure Supply Chain with Kubewarden: securing Kubewarden policies
```

Comment:
Selected this title, because I suppose we will have a follow-up `Secure Supply Chain with Kubewarden: securing containers`, with the new policy.